src/components/styled-select/styled-select.module.scss

```
@@ -0,0 +1,206 @@
+@use "src/tokens/index" as *;
+@use "../select/popup.module";
+
+.root {
+	--styled-select_picker-min-width: 192px;
+
+	position: relative;
+	display: grid;
+	grid-template-columns:
+		minmax(0, 1fr)
+		calc(
+			var(--btn_regular_icon_size_default) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)